gammapy/time/exptest.py

```diff
@@ -0,0 +1,41 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+
+__all__ = [
+    'exptest_for_run',
```

Comment:
Would it be OK for you to just rename this function from `exptest_for_run` to `exptest`?
The term "run" probably won't be used in CTA ... if you want to keep it use `exptest_for_observation`.
But I'd prefer to just have the simpler name `exptest` for this function.